pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -2357,6 +2368,78 @@ func (l *LibvirtDomainManager) parseFSDisks(fsDisks []api.FSDisk) []v1.VirtualMa
 	return disks
 }
 
+func (l *LibvirtDomainManager) linkImageVolumeFilePaths(vmi *v1.VirtualMachineInstance) error {
```

Comment:
nit: since the function is part of a pretty complex flow, I'd throw a little comment summarizing its operation.

Reply:
Done